pkgs/applications/editors/emacs/elisp-packages/manual-packages/codeium/default.nix

```diff
@@ -19,17 +20,19 @@ melpaBuild {
 
   patches = [
     (substituteAll {
-      src = ./codeium.el.patch;
-      codeium = "${pkgs.codeium}/bin/codeium_language_server";
+      src = ./0000-set-codeium-command-executable.patch;
+      codeium = lib.getExe' codeium "codeium_language_server";
     })
   ];
 
+  passthru.updateScript = unstableGitUpdater { };
```

Comment:
This packages uses a stable release.  So `unstableGitUpdater` does not work.  You can use `gitUpdater` instead.